scripts/oncallAutoAssignIssue.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+autoIssue() {
+  assigneeName=$1
+  token=$2
+  issueNum=$3
+  # Check if the user can be assigned
+  echo "Attempting to assign user $assigneeName to issue $issueNum"
+  checkAccess=$(curl --silent --output /dev/null --write-out "%{http_code}" -L -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $token" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/repos/microsoft/Oryx/assignees/$assigneeName)
+  if [[ "$checkAccess" =~ ^2 ]]; then
+    res=$(curl --silent --output /dev/null --write-out "%{http_code}" -X POST "https://api.github.com/repos/microsoft/Oryx/issues/$issueNum/assignees" -H "Accept: application/vnd.github+json" -H "Authorization: Bearer $token" -d "{\"assignees\": ["\"$assigneeName\""]}")
+    # Check for curl response status
+    if [[ "$res" =~ ^2 ]]; then
+      exit 0
+    else
+      echo "Error: Failed to assign isssue $issueNum. Check token and issue status."
+      exit 1
+    fi
+  else
+    echo "Error: Engineer Access denied."
+    return 1
+  fi
+}
+
+tryAssignIssueToCustomOncall() {
+  today=$1
+  token=$2
+  issue=$3
+  input="$CUSTOM_ONCALL_ROTATION"
+  input="${input//$'\n'/''}"
+  input="${input//$'\r'/';'}"
+  if [ -z "$input" ]; then
+    return 1
+  fi
+  # Custom oncall rotation defined in repo variable. Using format:
+  # {github-username},{start-date},{end-date}
+
+  # Devide each entry by ";"
```

Comment:
nit: typo, `devide` --> `divide`

Same comment down below in the next in-line comment you've made